components/Availability.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+import { availabilityOptions, useFilterContext } from '../context/FilterContext';
+import FieldsetGroup from './FieldsetGroup';
+import FilterToggle from './FilterToggle';
+
+export default function Availability() {
+  const [checkedValue, setCheckedValue] = useState(availabilityOptions.AVAILABLE);
```

Comment:
That was originally because when I'd used `availability` from context, the state was updating, but the DOM wasn't. Although something must have been wonky in my original implementation because it's working fine with this edit now.